source/features/merged-pr-first-published-tag.tsx

```diff
@@ -0,0 +1,66 @@
+import React from 'dom-chef';
+import cache from 'webext-storage-cache';
+import select from 'select-dom';
+import TagIcon from 'octicon/tag.svg';
+import * as pageDetect from 'github-url-detection';
+
+import features from '../libs/features';
+import fetchDom from '../libs/fetch-dom';
+import {getRepoURL} from '../libs/utils';
+import observeElement from '../libs/simplified-element-observer';
+
+const getFirstTag = cache.function(async (commit: string): Promise<string | undefined> => {
+	const firstTag = await fetchDom<HTMLAnchorElement>(`/${getRepoURL()}/branch_commits/${commit}`, 'ul.branches-tag-list li:last-child a');
+
+	return firstTag?.textContent as string;
+}, {
+	cacheKey: ([commit]) => `first-tag:${getRepoURL()}:${commit}`
+});
+
+async function addTag(discussionHeader: HTMLElement): Promise<void | false> {
+	const mergeCommit = select(`.TimelineItem.js-details-container.Details a[href^="/${getRepoURL()}/commit/" i] > code.link-gray-dark`)!.textContent!;
+	const tagName = await getFirstTag(mergeCommit);
+
+	if (!tagName) {
+		return;
+	}
+
+	discussionHeader.parentElement!.append(
+		<> • <TagIcon className="mx-1 text-gray-light v-align-middle"/>
```

Comment:
`.append` can accept multiple elements, no need for an intermediate `<>`

Reply:
When I did the `•` it was not working

Reply:
• should be a string

```js
.append(
	' • ',
	<TagIcon .../>,
	<a>...</a>
)
```

Reply:
`Parsing error: JSX expressions must have one parent element.XO`

Reply:
`Parsing error: JSX expressions must have one parent element.XO`

Reply:
Notice the commas...

Reply:
Sorry I misunderstood you. Fix it!
Thanks